qbit_manage.py

```diff
@@ -149,6 +149,15 @@
     default=False,
     help="Use this to skip cleaning up Recycle Bin/Orphaned directory.",
 )
+parser.add_argument(
+    "-svc",
+    "--skip-qb-version-check",
+    dest="skip_qb_version_check",
+    action="store_true",
+    default=False,
+    # help="Bypass qBittorrent/libtorrent version compatibility check. You run the risk of undesirable behavior and will receive no support.",
```

Comment:
Can you separate this into two lines? flake8 seems to be failing because the line is too long.